Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,15 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+console.log(`QTY     ITEM                TOTAL`);
+let Total = 0;
```

Comment:
What is naming convention for variable names? When it is recommended to use Capital case and lower case? It is related to your: let Total, and const Cost variables.